Frontend/src/Components/CreateListing/CreateListing.jsx

```diff
@@ -5,12 +5,18 @@ import PhotoSection from './PhotoSection';
 import DescriptionSection from './DescriptionSection';
 import InfoSection from './InfoSection';
 
+const backendApi = import.meta.env.VITE_BACKEND_ADDRESS;
```

Comment:
nit: for constants, we name them in all caps with _ in between words, so for this we would want to name it API_KEY or something like that